2019-11-01 23:03:01,177	INFO	main1	<module>	Line:11	app started
2019-11-01 23:03:01,193	WARNING	_internal	_log	Line:122	 * Debugger is active!
2019-11-01 23:03:01,245	INFO	_internal	_log	Line:122	 * Debugger PIN: 185-856-785
2019-11-01 23:03:01,269	INFO	_internal	_log	Line:122	 * Running on http://192.168.2.210:9100/ (Press CTRL+C to quit)
2019-11-01 23:03:04,713	INFO	_internal	_log	Line:122	192.168.2.210 - - [01/Nov/2019 23:03:04] "GET /ProblemSet/init/497925dd-2143-413d-83f9-7edfc2f7ba2e/ww HTTP/1.1" 200 -
2019-11-01 23:03:04,808	INFO	_internal	_log	Line:122	192.168.2.210 - - [01/Nov/2019 23:03:04] "GET /ProblemSet/answer/497925dd-2143-413d-83f9-7edfc2f7ba2e/5 HTTP/1.1" 500 -
2019-11-01 23:03:08,281	INFO	_internal	_log	Line:122	192.168.2.210 - - [01/Nov/2019 23:03:08] "GET /Login/logout/497925dd-2143-413d-83f9-7edfc2f7ba2e/test HTTP/1.1" 200 -
2019-11-01 23:03:44,460	INFO	_internal	_log	Line:122	 * Detected change in 'C:\\development\\examsystem\\services\\daos\\problemsetDAO.py', reloading
